src/features/DebugSession.ts

```diff
@@ -2,9 +2,10 @@
  * Copyright (C) Microsoft Corporation. All rights reserved.
  *--------------------------------------------------------*/
 
+import path = require("path");
```

Comment:
```suggestion
import * as path from "path";
```

Reply:
Removed this since we weren't using `path` anywhere in the file.